panimg/image_builders/tiff.py

```diff
@@ -36,6 +36,13 @@
     pyvips = False
 
 
+DICOM_WSI_STORAGE_ID = "1.2.840.10008.5.1.4.1.1.77.1.6"
+
+
+def format_error(message: str) -> str:
+    return f"Tif image builder: {message}"
```

Comment:
```suggestion
    return f"TIFF image builder: {message}"
```

abbreviation of Tag Image File Format.